build.py

```diff
@@ -2281,12 +2286,12 @@ def enable_all():
                         if not skip:
                             tagged_be_list.append(tagged_backend(be, ver))
                         skip = False
+            if (be == 'adsbrain'):
+                github_organization = 'https://github.com/feihugis'
 
             # If armnn_tflite backend, source from external repo for git clone
             if be == 'armnn_tflite':
                 github_organization = 'https://gitlab.com/arm-research/smarter/'
-            else:
-                github_organization = FLAGS.github_organization
```

Comment:
why delete it?

Reply:
move it to line 2278 to make `github_organization` more accessible.